### PR TITLE
Small doc fix in Getting-Started.md

### PR DESCRIPTION
**Pull request by nwoltman**
~~Remove unnecessary log line from an example since the error handler will end up logging the error. Also `req` was spelled wrong and isn't a variable in the function.~~

Rewrite the handler in the example to be short and concise and to remove an incorrect line.

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by nwoltman** [comment]
Hmm, looks like tests are failing because of `find-my-way@1.10.2`

**Comment by delvedor** [comment]
> Hmm, looks like tests are failing because of find-my-way@1.10.2

I'm working on this.

File: docs/Getting-Started.md [hunk 1]
```diff
@@ -159,7 +159,6 @@ async function routes (fastify, options) {
     try {
       return await collection.findOne({ id: request.params.id })
     } catch (err) {
-      reg.log.error(err)
       return new Error('Something went wrong')
     }
```
Comment (mcollina) [inline docs/Getting-Started.md]: Imho we should do something different in this route, change what is returned by `collection.findOne()`  for example. This try-catch is pointless.

File: docs/Getting-Started.md [hunk 2]
```diff
@@ -155,13 +155,8 @@ async function routes (fastify, options) {
...
 
-  fastify.get('/search/:id', async (request, reply) => {
-    try {
-      return await collection.findOne({ id: request.params.id })
-    } catch (err) {
-      reg.log.error(err)
-      return new Error('Something went wrong')
-    }
+  fastify.get('/search/:id', (request, reply) => {
+    return collection.findOne({ id: request.params.id })
```
Comment (mcollina) [inline docs/Getting-Started.md]: the point of this example was to show the usage with an async function, hence the `await`. The try-catch was silly, but we need to keep the `await`  and do something with the result of `findOne`. I know that the promise is implicit, but still it'd be helpful to show.

**Pull request merged**